packages/server/lib/controllers/v1/plan/postPlanExtendTrial.ts

```diff
@@ -0,0 +1,45 @@
+import db from '@nangohq/database';
+import { AnalyticsTypes, TRIAL_DURATION, analytics, updatePlan } from '@nangohq/shared';
+import { flagHasPlan, requireEmptyBody, requireEmptyQuery, zodErrorToHTTP } from '@nangohq/utils';
+
+import { asyncWrapper } from '../../../utils/asyncWrapper.js';
+
+import type { PostPlanExtendTrial } from '@nangohq/types';
+
+export const postPlanExtendTrial = asyncWrapper<PostPlanExtendTrial>(async (req, res) => {
+    const emptyQuery = requireEmptyQuery(req, { withEnv: true });
+    if (emptyQuery) {
+        res.status(400).send({ error: { code: 'invalid_query_params', errors: zodErrorToHTTP(emptyQuery.error) } });
+        return;
+    }
+
+    const emptyBody = requireEmptyBody(req);
+    if (emptyBody) {
+        res.status(400).send({ error: { code: 'invalid_body', errors: zodErrorToHTTP(emptyBody.error) } });
+        return;
+    }
+
+    const { plan } = res.locals;
+    if (!flagHasPlan) {
+        res.status(400).send({ error: { code: 'feature_disabled' } });
+        return;
+    }
+
+    if (!plan || !plan.trial_start_at || !plan.trial_end_at || (plan.name !== 'free' && plan.name !== 'test')) {
```

Comment:
`plan.name !== 'test'` ??

Reply:
ah yes leftover, I initially created "test" plan in vitest